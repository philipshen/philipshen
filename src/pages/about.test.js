import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import AboutPage from './about.js'

const renderer = new ShallowRenderer()

describe('About', () => {
  
  it('Matches snapshot', () => {
    const tree = renderer.render(<AboutPage />)
    expect(tree).toMatchSnapshot()
  })

})