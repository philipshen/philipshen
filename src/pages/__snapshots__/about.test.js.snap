// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Matches snapshot 1`] = `
<Layout
  currentHeaderSection={4}
  shouldHideHeader={true}
  spacer={true}
>
  <ForwardRef>
    <h1>
      About
    </h1>
    <div>
      <h4>
        Who made this site?
      </h4>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "Phil",
          }
        }
      />
    </div>
    <div>
      <h4>
        How is this site so dang fast?
      </h4>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "This site is powered by <a href=\\"https://gatsbyjs.org\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Gatsby</a>!",
          }
        }
      />
    </div>
    <div>
      <h4>
        Why does this site look so nice?
      </h4>
      <p
        dangerouslySetInnerHTML={
          Object {
            "__html": "This site was made with the help of <a href=\\"https://styled-components.com\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">styled-components</a> and <a href=\\"https://material-ui.com/style/icons/\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Material Icons</a>!",
          }
        }
      />
    </div>
  </ForwardRef>
</Layout>
`;
